README.md

```diff
@@ -28,7 +28,7 @@ The easiest way to get started with cljfmt is to add the lein-cljfmt
 plugin to your [Leiningen][] project map:
 
 ```clojure
-:plugins [[lein-cljfmt "0.1.1"]
+:plugins [[lein-cljfmt "0.1.2"]
```

Comment:
Pull requests should contain only the changes necessary to implement the described patch. You'll need to remove things like version number and whitespace changes.